src/platform/backends/lxd/lxd_virtual_machine.cpp

```diff
@@ -209,12 +211,19 @@ void mp::LXDVirtualMachine::stop()
         return;
     }
 
-    request_state("stop");
+    if (present_state == State::starting)
+    {
+        stop_options.insert("force", true);
+        forced_shutdown = true;
+    }
+
+    request_state("stop", stop_options);
```

Comment:
Add a TODO relating to #1909, please?

Reply:
Hmm, this particular issue seems mostly orthogonal to #1909.  The problem I'm trying to fix here is that LXD will ignore a `stop` on a VM if it's not completely booted, ie, in the `starting` state unless a `force` option is used.

It looks to me that #1909 is trying to solve the issue of being able to shut down a "stuck" VM, regardless of backend used.

That said, fixing #1909 shouldn't have any bearing on this change here and so we shouldn't add that TODO.

Reply:
Well, yeah, but when we introduce a `force_stop()` or so on the backend interface, this could call that instead, rather than reimplementing it?

Reply:
Ah, ok, I see your point there.  Ok, I'm convinced :wink:

Reply:
Done